adapters/handlers/graphql/local/aggregate/hybrid_search.go

```diff
@@ -68,8 +68,16 @@ func hybridOperands(classObject *graphql.Object,
 			Type:        graphql.NewList(graphql.String),
 		},
 		"properties": &graphql.InputObjectFieldConfig{
-			Description: "Target vectors",
-			Type:        graphql.NewList(graphql.String),
+			Description: "Properties to search",
+			Type:        graphql.Int,
```

Comment:
I think that this is a breaking change? can we introduce this change in some other way?